src/libs/AxelarQueryAPI.ts

```diff
@@ -92,11 +92,13 @@ interface HopParams {
    */
```

Comment:
gasLimit shouldn't be needed for transfers. A gas limit can be provided for the contract execution